x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/huggingface/HuggingFaceBaseService.java

```diff
@@ -0,0 +1,109 @@
+/*
```

Comment:
A base class for the hugging face implementations. If we can remove the elser one we can just move this directly into `HuggingFaceService`.